elastic-container.sh

```diff
@@ -3,20 +3,38 @@ set -o pipefail
 
 ipvar="0.0.0.0"
 
+# These should be set in the .env file
+declare LinuxDR
+declare WindowsDR
+declare MacOSDR
+
+# Ignore following warning
+# shellcheck disable=SC1091
 . .env
 
+declare COMPOSE
+if docker compose 2>/dev/null; then
+  COMPOSE="docker compose"
+elif command -v docker-compose >/dev/null; then
```

Comment:
```suggestion
if docker compose >/dev/null 2>&1; then
  COMPOSE="docker compose"
elif command -v docker-compose >/dev/null 2>&1; then
```

Reply:
I don't want to throw this completely away, so I'll move it until after the verbose check